packages/picasso.js/src/core/chart-components/ref-line/refline.js

```diff
@@ -182,6 +188,7 @@ const refLineComponent = {
 
   render() {
     let settings = this.settings;
+    let slopeLine;
```

Comment:
Why is `slopeLine` declared here? Can it not be declared in the scope where it's used?